content/news/speaker-support.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Speaker Support for PyLadiesCon 2024"
+meta_title: "Speaker Support for PyLadiesCon 2024"
+description: "We have speaker support for those of you who need a bit more reassurance and encouragement!
+"
+date: 2024-08-28T05:00:00Z
+categories: ["Blog Post",]
+author: "Cheuk"
+tags: ["python", "community", "conference", "public speaking", "pyladies"]
+draft: false
+layout: news
+---
+
+Since there are only 2 weeks until the [Call for Propocal](https://pretalx.com/pyladiescon-2024/cfp) closes, we do not want to you miss this speaking opportunity. Do you still need some encouragement in submitting a proposal? Maybe you need a bit more help? Don't worry! We have speaker support for those of you who need a bit more reassurance and encouragement!
+
+## Support for new speakers
+
+If you are new to public speaking and would love to get a friendly mentor to help you prepare your submission or your talk when you are accepted. Please contact us at [pyladiescon@pyladies.com](mailto:pyladiescon@pyladies.com) with the title **Looking for speaker mentor** We will try our best to find a mentor for you. All you need to do is tell us a bit about yourself (where you are based, what language you would like to speak with your mentor and what do you need help with) so that we know how we can meet your needs.
+
```

Comment:
```suggestion

One of our community members, Mariatta, has set aside her time for PyLadiesCon Speaker Support. You can book a time with her through [her website](https://mariatta.ca/posts/pyladiescon-speaker-support/).
```

Reply:
Either this paragraph, or the suggestion below. (we don't need both)

Reply:
I won't recommend you put your link there, I am a bit worry that you may get over work if everyone tried to book a time with you. I think we should let everyone email us (or maybe fill in a form?) and we will assign them